src/main/java/me/svdev/mscategory/dao/entity/Category.java

```diff
@@ -0,0 +1,53 @@
+package me.svdev.mscategory.dao.entity;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "categories")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false, unique = true)
```

Comment:
don't need @Column validations in Entity class would be better use this logic in liquibase